rust/Cargo.toml

```diff
@@ -19,7 +19,39 @@ members = [
   "ip-packet",
   "logging",
   "phoenix-channel",
-  "relay",
+  "relay/ebpf-shared",
+  "relay/ebpf-turn-router",
+  "relay/server",
+  "socket-factory",
+  "telemetry",
+  "tests/gui-smoke-test",
+  "tests/http-test-server",
+  "tun",
+]
+
+# Everything but relay/ebpf-turn-router
+default-members = [
```

Comment:
This is needed to make `cargo check/build` etc still work, despite us having a crate in the workspace the needs to be built for a certain target and without any feature enabled.